lib/ProductOpener/Config_off.pm

```diff
@@ -357,14 +357,35 @@ $page_size = 24;
 
 $google_analytics = <<HTML
 <!-- Global site tag (gtag.js) - Google Analytics -->
-<script async src="https://www.googletagmanager.com/gtag/js?id=UA-31851927-1"></script>
+<script async src="https://www.googletagmanager.com/gtag/js?id=G-HQX9SYHB2P&aip=1"></script>
```

Comment:
It's not possible to keep the same id so that we have continuous stats?

Reply:
No, this is a requirement of GA4
Moreover GA4 drastically decrease retention period to 2 monthes (but we could set 4 monthes) (@teolemon you are aware of it right ?)

Reply:
ah ok, well if it retains data for only a few months anyway, then this point becomes moot.

but that means that we can't even have very aggregated metrics across years? e.g. getting a graph with monthly users and page views over a period of several years?

Reply:
It seems it's only for exploration: if you want to slice and dice data accross users: https://support.google.com/analytics/answer/7667196?hl=fr

But the summary metrics are kept.

Reply:
ok, anyway there's no choice, so let's go for it